modules/v1_12_R1/src/main/java/net/countercraft/movecraft/compat/v1_12_R1/IWorldHandler.java

```diff
@@ -1,17 +1,16 @@
 package net.countercraft.movecraft.compat.v1_12_R1;
 
-import net.countercraft.movecraft.config.Settings;
-import net.countercraft.movecraft.utils.MathUtils;
 import net.countercraft.movecraft.MovecraftLocation;
 import net.countercraft.movecraft.Rotation;
-import net.countercraft.movecraft.utils.CollectionUtils;
 import net.countercraft.movecraft.WorldHandler;
+import net.countercraft.movecraft.config.Settings;
 import net.countercraft.movecraft.craft.Craft;
+import net.countercraft.movecraft.utils.CollectionUtils;
+import net.countercraft.movecraft.utils.MathUtils;
```

Comment:
don't modify this file

Reply:
I think it got modified after I pulled recent commits from the upstream version into this branch